### PR TITLE
Add org.pegasus_frontend.Pegasus

### PR DESCRIPTION
**Pull request by mmatyas**
Adds Pegasus (https://pegasus-frontend.org) to FlatHub.

### CODE REVIEW

File: org.pegasus_frontend.Pegasus.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  - --share=network
+
+modules:
+  - name: pegasus-frontend
+    buildsystem: qmake
+    builddir: true
+    config-opts:
+      - USE_SDL_GAMEPAD=1
+      - INSTALL_BINDIR=/app/bin
+      - INSTALL_ICONDIR=/app/share/icons/hicolor/128x128/apps
```
Comment (bilelmoussaoui) [inline org.pegasus_frontend.Pegasus.yml]: the metainfo requires 128 & 64 sizes.

File: org.pegasus_frontend.Pegasus.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
+app-id: org.pegasus_frontend.Pegasus
+runtime: org.kde.Platform
+runtime-version: '5.12'
```
Comment (bilelmoussaoui) [inline org.pegasus_frontend.Pegasus.yml]: Any reason for not using 5.13?

File: org.pegasus_frontend.Pegasus.yml (hunk 1 above)
Comment (mmatyas) [reply org.pegasus_frontend.Pegasus.yml]: You mean both 128 and 64? I see, will follow up with a patch.

File: org.pegasus_frontend.Pegasus.yml (hunk 2 above)
Comment (mmatyas) [reply org.pegasus_frontend.Pegasus.yml]: 5.13 dropped the Canvas3D module, but its alternative, Qt3D has been [found to be broken](https://bugreports.qt.io/browse/QTBUG-66587) in certain platforms in the past. At the moment Canvas3D support is still assumed, but I can upgrade to later Qt versions in the future.

**Comment by bilelmoussaoui** [comment]
bot, build org.pegasus_frontend.Pegasus

**Comment by mmatyas** [comment]
Ok, added a local icon file. In case I modify the upstream repo in the future to generate multiple different sized icons, would it be okay to edit/remove the relevant part of the manifest?

**Comment by LeandroStanger** [comment]
bot, build org.pegasus_frontend.Pegasus

**Comment by mmatyas** [comment]
Fixed the issues in the upstream source, running the validator locally now produces no errors for me.

**Comment by bilelmoussaoui** [comment]
bot, build org.pegasus_frontend.Pegasus

**Comment by bilelmoussaoui** [comment]
Anyone else from the maintainers needs write access to the repository once created?

**Comment by mmatyas** [comment]
No, I'm planning to manage the repository only.

**Pull request closed without merging**